src/mmu/paging.c

```diff
@@ -309,3 +309,34 @@ void map(uint64_t page_number, uint64_t flags) {
 
     map_page_to_frame(page_number, frame, flags);
 }
+
+void map_multiple(uint64_t start_page_number, uint32_t number_of_pages, uint64_t flags) {
+
+    for (uint32_t i = 0; i < number_of_pages; i++) {
+
+        map(start_page_number + i, flags);
+    }
+}
+
+void unmap_multiple(uint64_t start_page_number, uint32_t number_of_pages) {
+
+    for (uint32_t i = 0; i < number_of_pages; i++) {
+
+        unmap(start_page_number + i);
+    }
+}
+
+uint32_t paging_amount_for_byte_size(uint64_t start_address, uint64_t byte_size) {
```

Comment:
Is this used somewhere? If not, I suggest we do not introduce it in this patch.

Reply:
This is definitely going to be used a lot since it's the easiest way of finding the pages to `map_multiple`. It's already used in #35.

Reply:
Ok, I guess I was trying to say that we should keep patches small and only add what is actually used in the proposed patches.

Reply:
I would say that it makes sense to have some functionality that we know are universally useful and needed regardless of what the PR actually does. But that would be very limited things. Finding how many pages you must map sounds naturally necessary for a new multi-mapping paging API.

Reply:
Sure but it could be added in a different patch :)